jobbergate-api/jobbergate_api/apps/job_submissions/routers.py

```diff
@@ -90,6 +91,16 @@ async def job_submission_create(
             detail=message,
         )
 
+    try:
+        job_properties = get_job_properties_from_job_script(
+            job_submission.job_script_id, **job_submission.execution_parameters.dict(exclude_unset=True)
+        )
+        create_dict["execution_parameters"] = job_properties.dict(exclude_unset=True)
```

Comment:
I know the variable creation comes from other place but let's agree a variable named `create_dict` isn't that descriptive. It indicates an action, quite weird I'd say.

Reply:
solved in [cb83d36](https://github.com/omnivector-solutions/jobbergate/pull/193/commits/cb83d36dd346db4174308fad60a9f9480cae0e75)